runtime/lua/vim/shared.lua

```diff
@@ -12,7 +12,7 @@ local vim = vim or {}
 --- same functions as those in the input table. Userdata and threads are not
 --- copied and will throw an error.
 ---
----@param orig Table to copy
+---@param orig table to copy
```

Comment:
The annotation format is `@param {name} {type} {description}`

```suggestion
---@param orig table Table to copy
```

Reply:
Technically deepcopy isn't just for `table`. It works on everything but `cdata` & `coroutine` I think.

Reply:
“Deep copying” a number, boolean, string, or nil is meaningless, so we don’t need to include those.

Reply:
Ok